features/yard-doctest.feature

```diff
@@ -785,3 +785,26 @@ Feature: yard doctest
       """
       app/app.rb:4:in `foo'
       """
+  
+  Scenario: Expecting `nil` does not issue deprecation warning
```

Comment:
Instead of adding another scenario, can you please extend https://github.com/p0deje/yard-doctest/blob/24beb616afb6acebb6038739c09f5e4f297cd720/features/yard-doctest.feature#L186 and just add a verification that:

```cucumber
And the output should not contain "DEPRECATED"
```

Reply:
This way we might be able to catch other types of deprecations in future.

Reply:
Yeah, that's much better. Made the change and pushed a new commit to this branch.